tests/test_vec_normalize.py

```diff
@@ -76,6 +76,53 @@ def test_vec_env(tmpdir):
     check_vec_norm_equal(norm_venv, deserialized)
 
 
+def _make_warmstart_cartpole():
+    """Warm-start VecNormalize by stepping through CartPole"""
+    venv = DummyVecEnv([lambda: gym.make("CartPole-v1")])
+    venv = VecNormalize(venv)
+    venv.reset()
+    venv.get_original_obs()
+
+    for _ in range(100):
+        acts = [venv.action_space.sample()]
+        venv.step(acts)
+    return venv
+
+
+def test_get_original():
+    venv = _make_warmstart_cartpole()
+    for _ in range(3):
+        acts = [venv.action_space.sample()]
```

Comment:
I know the old code uses `acts` and `rews` but please use `actions` and  `rewards`,
it is not that much longer but it is clearer